app/src/main/java/org/sopt/at/MyActivity.kt

```diff
@@ -0,0 +1,36 @@
+package org.sopt.at
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import org.sopt.at.ui.theme.ATSOPTANDROIDTheme
+
+class MyActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val userId = intent.getStringExtra("userId") ?: ""
+        enableEdgeToEdge()
+        setContent {
+            ATSOPTANDROIDTheme {
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    MyView(Modifier.padding(innerPadding), userId)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun MyView(modifier: Modifier, userId: String) {
```

Comment:
인자가 2개 이상이 되는 순간부터 줄바꿈을 통해 가독성을 높여주는게 좋습니다 !
```suggestion
fun MyView(
   modifier: Modifier,
   userId: String
 ) {
```
이런 식으로요 ~~

Reply:
또, 보통 수정자는 선택인자로 받습니다 !
```modifier: Modifier = Modifier```
요런 느낌입니다이